packages/workflow/src/internals.ts

```diff
@@ -181,7 +241,7 @@ export interface ExternalCall {
   fnName: string;
   args: any[];
   /** Optional in case applyMode is ASYNC_IGNORED */
-  seq?: number;
+  seq?: string;
```

Comment:
Will this go back to being a number when we make the ids change?

Reply:
Yes